### DISCUSSION TITLE
Getting an error when trying to edit certain recipes

### DISCUSSION THREAD

**Discussion by edcous** [TandoorRecipes/recipes#274] [Q&A] [answered]
Hi! When trying to edit certain recipes I get an error:
`There was an error updating the recipe!{"steps":[{"ingredients":[{},{},{},{},{},{"unit":{"name":["This field may not be blank."]}},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]}]}`

A bit unsure why only certain recipes have this error, any ideas?

**Comment by vabene1111** [comment, marked as answer]
Yea this is a stupid error .. It resulted from me making fields not required that were required before and some parts of the application not working together properly. It mostly happens for imported recipes, i need to look into when exactly it happens and then fix this issue.

For now just make sure that each ingredient has a unit selected, that should do the trick.

**Comment by edcous** [comment]
On other recipes I get the error `An error occurred while trying to import this recipe! <!doctype html> <html lang="en"> <head> <title>Server Error (500)</title> </head> <body> <h1>Server Error (500)</h1><p></p> </body> </html>` as well.

**Reply by vabene1111** [reply]
this is not related to the same problem because its the URL importer and not the recipe editing function. Please open an issue with an exact problem description and this case example URL for me to reproduce it.

**Reply by edcous** [reply]
Ok, I'll do this in a few minutes.

**Reply by edcous** [reply]
Done, #276
